### DISCUSSION TITLE
Better support for long file names, increase information density per pixel in file list

### DISCUSSION THREAD

**Discussion by aisbergde** [gsantner/markor#1559] [Ideas] [answered]
Since I use longer filenames for better searching, they are regularly truncated. Switching from portrait mode to landscape mode brings a little help, but then only 4 net lines are displayed.

I would like it if I, as a user, could set some things to allow me to read longer filenames.

- Font size
- bold or normal
- less loss due to the icon column on the left
- line break

Overall the information density per pixel in the file list is very low.

I can also add some screenshots of other programs.

**Comment by aisbergde** [comment, marked as answer]
Nearly all file explorers can show long file names. Most are using line wrap, some are using a Treadmill display (Laufband Anzeige)
![Screenshot_2022-01-09-10-45-28-752_net.gsantner.markor.jpg](https://user-images.githubusercontent.com/9020187/148677568-f1a9629e-a588-43c1-94bf-9c945b6be099.jpg)
![Screenshot_2022-01-09-10-46-22-368_net.gsantner.markor.jpg](https://user-images.githubusercontent.com/9020187/148677573-f93da51a-6f60-46f1-9345-39a5a62304fb.jpg)
![Screenshot_2022-01-09-10-47-50-577_com.cxinventor.file.explorer.jpg](https://user-images.githubusercontent.com/9020187/148677583-e3c6ea9f-53e8-4e00-b9ec-50dcb8751d42.jpg)
![Screenshot_2022-01-09-10-48-47-681_files.fileexplorer.filemanager.jpg](https://user-images.githubusercontent.com/9020187/148677587-45d3994d-d1c5-4341-afcf-f0e13a12fd69.jpg)
![Screenshot_2022-01-09-10-49-50-763_com.ghisler.android.TotalCommander.jpg](https://user-images.githubusercontent.com/9020187/148677594-23209d7b-a701-449c-b0d0-3d6f03d9f1f8.jpg)
![Screenshot_2022-01-09-10-51-23-870_com.mi.android.globalFileexplorer.jpg](https://user-images.githubusercontent.com/9020187/148677601-0648f3e0-ff7e-4261-8c7b-4ad661663a5d.jpg)
![Screenshot_2022-01-09-10-54-54-290_com.amaze.filemanager.jpg](https://user-images.githubusercontent.com/9020187/148677609-4b976ecd-fd1b-4b43-a033-8f9c13b13ce1.jpg)

**Reply by gsantner** [reply]
I thought about allowing multi-line expansion of filenames for a while, but didn't make a decision.

Now made the change, with the next update (v2.8.5 / v2.9.0):
The folder/filenames will then be shown fully, expanded to multiple lines.
